blocks/tags-content-source-block/sources/tags-api.js

```diff
@@ -1,26 +1,38 @@
-const make404 = () => {
-	const error = new Error("Not found");
-	error.statusCode = 404;
-	throw error;
+import axios from "axios";
+import { ARC_ACCESS_TOKEN, CONTENT_BASE, RESIZER_APP_VERSION } from "fusion:environment";
+
+import signImagesInANSObject from "@wpmedia/arc-themes-components/src/utils/sign-images-in-ans-object";
+import { fetch as resizerFetch } from "@wpmedia/signing-service-content-source-block";
+
+const params = {
+	slug: "text",
 };
 
-export default {
-	resolve({ slug }) {
-		return `/tags/v2/slugs?slugs=${slug}`;
-	},
-	params: {
-		slug: "text",
-	},
-	transform: (data) => {
-		if (!data || !data.Payload) {
-			make404();
-		}
+const fetch = ({ slug: slugs = "" }, { cachedCall }) => {
+	const urlSearch = new URLSearchParams({ slugs });
 
-		const hasValues = data.Payload.some((ele) => !!ele);
-		if (hasValues) {
-			return data;
-		}
+	return axios({
+		url: `${CONTENT_BASE}/tags/v2/slugs?${urlSearch.toString()}`,
+		headers: {
+			"content-type": "application/json",
+			Authorization: `Bearer ${ARC_ACCESS_TOKEN}`,
+		},
+		method: "GET",
+	})
+		.then(signImagesInANSObject(cachedCall, resizerFetch, RESIZER_APP_VERSION))
+		.then(({ data }) => {
+			if (data?.Payload?.some((ele) => !!ele)) {
+				return data;
+			}
 
-		return make404();
-	},
+			const error = new Error("Not found");
+			error.statusCode = 404;
+			return Promise.reject(error);
+		});
+};
+
+export default {
+	fetch,
+	params,
+	schemaName: "ans-feed",
```

Comment:
Tags API is does not return ANS items. Setting this `schemaName` here will enable this content source to be available within some of our blocks and would not work for the user. We can remove this `schemaName` value and leave off as per previous implementation

Reply:
@matthewroach : given that this is not an ANS item, do we really need to check for images and add resizer auth?  (that goes for some of the others as well)

Reply:
Probably not no - we should only need to update content sources that are doing the resizer logic today